README.md

```diff
@@ -42,6 +42,10 @@ The following environment variables are required:
 | HTTP_CLIENT_MAX_CLIENT_PER_ROUTE | 20 | Used for Pooling connection manager; for further information, see https://hc.apache.org/httpcomponents-client-ga/tutorial/html/connmgmt.html |
 | HTTP_CLIENT_VALIDATE_AFTER_INACTIVITY | 0 | Used for Pooling connection manager; for further information, see https://hc.apache.org/httpcomponents-client-ga/tutorial/html/connmgmt.html |
 | HTTP_CLIENT_CONNECTION_TIMEOUT | 30000 | 30 seconds, Used for Pooling connection manager; for further information, see https://hc.apache.org/httpcomponents-client-ga/tutorial/html/connmgmt.html |
+| HTTP_CLIENT_CONNECTION_DRAFTS_TIMEOUT | 200 | 200 milliseconds, Used for Pooling connection manager for draft store; for further information, see https://hc.apache.org/httpcomponents-client-ga/tutorial/html/connmgmt.html |
+| DRAFT_STORE_URL | - | Base URL for Draft Store API service. `http://localhost:8800` for the dockerised local instance. |
+| DRAFT_ENCRYPTION_KEY | - | Draft encryption key. The encryption key used by draft store to encrypt documents with. |
+| DRAFT_MAX_STALE_DAYS | - | Number of days after which the saved draft will be deleted if unmodified. |
```

Comment:
can we call this DRAFT_TTL?

Reply:
Let's keep it same as draft store. Also it has the DAYS in it so more descriptive.

Reply:
if you want to have days indication we could call it DRAFT_TTL_DAYS. I'm insisting because these type of properties are universally called TTL. We should really align to that